Backend/routes/auth.py

```diff
@@ -10,8 +10,10 @@ def sign_in_route():
 @sign_up.route('/sign_up', methods=['POST'])
 def sign_up_route():

```

Comment:
Pick one:
1. separate endpoints for each db check and the final call and implement the other changes
2. no separation, you run the check all three times but you build the query string:
qs = 'some query stuff where'
if email:
  qs += f'email={email}'
if phoneNumber:
  qs += f'phone_number={phoneNumber}'

Reply:
basically #2 is make it dynamic so your ifs arent as ugly